testsuite/smoke-test/src/test_utils.rs

```diff
@@ -49,8 +49,9 @@ pub fn compare_balances(
 pub fn setup_swarm_and_client_proxy(
     num_nodes: usize,
     node_index: usize,
+    enable_websocket: bool,
```

Comment:
This is a fairly large change for something that should be specific to a single test. Lets drop this commit and instead just enable websockets in the single test that needs it. similarly to how this test tweaks a config and then launches the swarm: https://github.com/diem/diem/blob/main/testsuite/smoke-test/src/consensus.rs#L64-L79